packages/cli/generation/ir-migrations/src/migrations/v28-to-v27/__test__/migrateFromV28ToV27.test.ts

```diff
@@ -0,0 +1,14 @@
+import { AbsoluteFilePath, join, RelativeFilePath } from "@fern-api/fs-utils";
+import { createMigrationTester } from "../../../__test__/utils/createMigrationTester";
+import { V28_TO_V27_MIGRATION } from "../migrateFromV28ToV27";
+
+const runMigration = createMigrationTester(V28_TO_V27_MIGRATION);
+
+describe("migrateFromV28ToV27", () => {
+    it("maps boolean literals to boolean types", async () => {
+        const migrated = await runMigration({
+            pathToFixture: join(AbsoluteFilePath.of(__dirname), RelativeFilePath.of("./fixtures/literal")),
+        });
+        expect(migrated).not.toBe(null);
```

Comment:
This test could do a lot more (i.e. actually verify that the properties were mapped into a `PrimitiveType.Boolean` type reference. In its current state it's just verifying that migration runs without failure.

Reply:
It would be nice to do a little more verification here just because this logic will run "forever".

Reply:
Great call, done